dit.py

```diff
@@ -2,41 +2,71 @@
 # -*- coding: utf-8 -*-
 #
 # Author:        Filipe L B Correia <filipelbc@gmail.com>
-# Last Change:   2016 Ago 27 19:06:00
+# Contributor:   Daniel Moraes <daniel.b.moraes@gmail.com>
 #
 # About:         Command line work time tracking and todo list
 #
 # =============================================================================
 
 """
-Usage: dit [--verbose] [--directory "path"] <command>
+Usage:
 
-    new "id" [--group "id"] [--subgroup "id"]
+  dit [--verbose, -v] [--directory, -d "path"] <command>
 
-    workon [--new] "id" [--group "id"] [--subgroup "id"]
+  --directory, -d
+    Specifies the directory where the tasks are stored. Defaults to '~/.dit'.
 
-    halt
+  --verbose, -v
+    Prints detailed information of what is done.
 
-    switchto [--new] "id" [--group "id"] [--subgroup "id"]
+  --help, -h
+    Prints this message and quits.
 
-    conclude ["id"]  [--group "id"] [--subgroup "id"]
+  <command>:
 
-    list [--group "id"] [--subgroup "id"] [--all]
+    new <name> [-d "description"]
+      Creates a new task.
 
-    status
+    workon <id> | --new, -n <name> [-d "description"]
+      Clocks in the specified task.
+      --new, -n
+        Same as 'new' followed by 'workon'.
+
+    halt [<id> | <name>]
+      Clocks out of the specified task or the current one.
+
+    switchto <id> | --new, -n <name> [-d "description"]
+      Same as 'halt' followed by 'workon'
+
+    conclude [<id> | <name>]
```

Comment:
Conclude only makes sense for a task. Make it clear that one can not pass the ID of a group or a subgroup.